libs/designsystem/src/lib/components/charts/shared/chart-js-service/chart-js.service.spec.ts

```diff
@@ -469,8 +469,8 @@ describe('ChartJSService', () => {
       });
     });
 
-    let chartTypesThatDestructivelyUpdate: ChartType[] = ['bar', 'column'];
-    let chartTypesThatUpdateNormally: ChartType[] = ['line'];
+    const chartTypesThatDestructivelyUpdate: ChartType[] = ['bar', 'column'];
+    const chartTypesThatUpdateNormally: ChartType[] = ['line'];
```

Comment:
This is autofixed by eslint. Cant seem to avoid that without going through hoops, so leaving as is even though it is not at all related.